MREUnityRuntime/MREUnityRuntimeLib/Messaging/Payloads/NetworkCommandPayloads.cs

```diff
@@ -353,26 +354,33 @@ public class CreateAnimation : NetworkCommandPayload
         public string AnimationName { get; set; }
 
         /// <summary>
-        /// The enumeration of animation key frames to set to the animation.
+        /// The enumeration of animation key frames to set to the animation. See <see cref="MWAnimationKeyframe"/>.
         /// </summary>
         public IEnumerable<MWAnimationKeyframe> Keyframes { get; set; }
 
         /// <summary>
-        /// The enumeration of animation events to set to the animation.
+        /// The enumeration of animation events to set to the animation. See <see cref="MWAnimationEvent"/>.
         /// </summary>
         public IEnumerable<MWAnimationEvent> Events { get; set; }
 
         /// <summary>
         /// The wrap mode of the animation. See <see cref="MWAnimationWrapMode"/>.
         /// </summary>
+        [JsonConverter(typeof(Converters.DashFormattedEnumConverter))]
         public MWAnimationWrapMode WrapMode { get; set; }
+
+        /// <summary>
+        /// (Optional) The initial time, speed, and enable state of the animation (all values also optional). See <see cref="MWSetAnimationStateOptions"/>.
+        /// </summary>
+        public MWSetAnimationStateOptions InitialState { get; set; }
     }
 
     /// <summary>
+    /// DEPRECATED
     /// App => Engine
     /// Payload for when the app wants to start an animation.
     /// </summary>
-    public class StartAnimation : NetworkCommandPayload
+    public class DEPRECATED_StartAnimation : NetworkCommandPayload
```

Comment:
Can this be versioned instead of marked deprecated? There might be more deprecated versions in the future.

Reply:
These deprecated messages have been deleted entirely from the protocol. There won't be any new versions of them. They remain here to support older MREs so they aren't immediately broken when the next Altspace release goes out. I'm thinking of the Stargate.

Whenever we bump the protocol version, these messages can be removed at that time.